### DISCUSSION TITLE
How  to understand the 'targetSequences' and 'readCount'  in final results with mixcr-4.4.1

### DISCUSSION THREAD

**Discussion by xmy1990** [milaboratory/mixcr#1414] [Q&A] [answered]
Thank you very much at first!

I have two issues with my final results of *.tsv  with mixcr-4.4.1:

**How  to understand the 'targetSequences' and 'readCount'  columns in  *.clones_TRB.tsv or *.clones_IGH.tsv ?**
I have found them in the doc:
readCount, Number of reads assigned to the clonotype https://mixcr.com/mixcr/reference/mixcr-export/?h=targetsequences#alignment-specific-fields
targetSequences, Export aligned sequences (targets), separated with comma https://mixcr.com/mixcr/reference/mixcr-export/?h=targetsequences#common-fields

**If it's convenient, please help me to explan them in common language!**

Thank you very much again for your help!

**Comment by mizraelson** [comment]
Hi,
`targetSequences` refers to the sequence used to assemble clones. This sequence depends on the preset that you use to analyze data, and it may vary from `CDR3` to the full `VDJRegion`. If you can share the command you used, I can tell you which assembling feature was used.
`readCount` is the number of sequencing reads with a certain clonal sequence, i.e., "how many times this clone occurred in the sample."

Sincerely,
Mark

**Comment by xmy1990** [comment]
Thank you very much ！

The command I used:
mixcr analyze exome-seq 
--species hsa \
--threads 8 \
$bam \
$prefix

As :https://mixcr.com/mixcr/reference/overview-analysis-overview/#cdr3-extension
Can I say: **targetSequences** is  the CDR3 in the left figure and  **nSeqCDR3** is the **CDR3 clustering**  in the  right figure.


Thank you very much again for your help!

**Comment by mizraelson** [comment, marked as answer]
For `exome-seq`, the clones are first assembled by `CDR3`, but later `mixcr assembleContigs` is used to assemble the longest possible clonal sequence. Because reads in `exome-seq` randomly cover the clonal sequence, for some clones, it will be possible to assemble only the `CDR3` sequence, while for others (more abundant), it may be possible to assemble a longer sequence. 
Thus,
- `targetSequences` in this case refers to the longest sequence that was possible to assemble for a particular clone.
- `nSeqCDR3` is the nucleotide sequence of the `CDR3` region.

For some clones, `targetSequences` equals `nSeqCDR3`.
Let me know if you have any more questions.

Sincerely,
Mark
